src/routes/news/components/__tests__/TunewsDetailsFooter.spec.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import React from 'react'
+import { shallow } from 'enzyme'
+import moment from 'moment-timezone'
+import TunewsDetailsFooter from '../TunewsDetailsFooter'
+
+describe('TunewsDetailsFooter', () => {
+  const language = 'en'
+  const date = moment.tz('2020-03-20 17:50:00', 'GMT')
+  const t = (key: ?string): string => key || ''
+
+  it('should render and match snapshot', () => {
+    expect(shallow(
+      <TunewsDetailsFooter
+        eNewsNumber='tun0000009902'
+        date={date}
+        language={language}
+        t={t}
+      />
+    )).toMatchSnapshot()
```

Comment:
It would be very good if this tests would assert fro actual behavior of the component. Like that it displays some items.